api-docs-slate/source/includes/_coin.md

```diff
@@ -1,10 +1,13 @@
-# Coin
+# bcoin - Coins
 Getting coin information via API.
 
 *Coin stands for UTXO*
 
 <aside class="info">
-You need to enable <code>index-address</code> in order to lookup coins by address(es).
+You need to enable <code>index-address</code> in order to lookup coins by address.<br>
```

Comment:
Maybe a note that this can't be enabled retroactively without a resync (there is a PR in review to allow this to happen in parallel to the chain)